holidayextras/brands
# Amend product list panels to fit better, button styling etc.

#### What does this PR do? (please provide any background)
https://hxshortbreaks.atlassian.net/browse/FP-1202

This adds some styling changes for the 'new' hotel panels. 
- Reduces the vertical size of the call to action buttons
- Changes the colour of the stars
- Crunches up some margins to save some spacing

#### How can this be tested?
`cd pattern-library && npm link ../brands`

Run this branch of brands and [this themeBlueprint PR](https://github.com/holidayextras/themeBlueprint/pull/1043)

Check that the buttons are as per screenshots
#### Any tech debt?
Nope
#### Screenshots / Screencast
**Before**
![screen shot 2017-08-21 at 08 40 18](https://user-images.githubusercontent.com/5682214/29508562-e48e652a-864c-11e7-8c37-2326d2db44b3.png)
**After**
![screen shot 2017-08-21 at 08 38 31](https://user-images.githubusercontent.com/5682214/29508563-e49a0f92-864c-11e7-8d2a-127c0242dd3c.png)

#### What gif best describes how you feel about this work?
![](https://media.giphy.com/media/bO50DnjvwJ3ZS/giphy.gif)
